www/errorreport.php

```diff
@@ -101,10 +101,21 @@
 // check that it looks like a valid email address
 if (!preg_match('/\s/', $email) && strpos($email, '@') !== false) {
     $replyto = $email;
-    $from = $email;
 } else {
     $replyto = null;
-    $from = 'no-reply@simplesamlphp.org';
+}
+
+$from = $config->getString('sendmail_from', null);
+if ($from === null || $from === '') {
+    $from = ini_get('sendmail_from');
+    if ($from === '' || $from === false) {
+        $from = 'no-reply@example.org';
```

Comment:
Lines 110-113 could perhaps be consolidated into:
```php
$from = ini_get('sendmail_from', 'no-reply@example.org');
```
?

Reply:
Nope, not unless `ini_get()` has an undocumented 2nd parameter...

Reply:
Right, I was confused with our own config get functions...